.ci/jenkins_job_build.sh

```diff
@@ -12,6 +12,10 @@ source "/etc/os-release"
 # Signify to all scripts that they are running in a CI environment
 [ -z "${KATA_DEV_MODE}" ] && export CI=true
 
+# CI environment on aarch64 only supports bare mental for now
+arch=$(uname -m)
+[ "${arch}" == "aarch64" ] && export BARE=true
```

Comment:
It might be better to `export BARE=true` in the Jenkins job setup scripts on the jenkins master, and/or even use a Jenkins slave label and see if we can get that exported into the environment.
That way this script stays more generic, and later if we do get nested VM ARM Jenkins slaves we don't have to come fix this script again - we  can just set the correct settings inside Jenkins itself.
That is what I do for the [`METRICS_CI`](https://github.com/kata-containers/tests/blob/master/.ci/jenkins_job_build.sh#L136) and [`JSON_URL`](https://github.com/kata-containers/tests/blob/master/metrics/lib/json.bash#L115) vars for instance.

Reply:
right~~Let's set it in the Jenkins job setup scripts on the jenkins master for  ARM CI. Since I don't have the perm, maybe you'd like to fix it? 😊

Reply:
sure, I can give jenkins access when needed.

Reply:
That's fine with me ;).